### ISSUE TITLE
enhancement: xpra subcommands should assume current instance where possible

### ISSUE THREAD

**Issue by karlkleinpaste** [Xpra-org/xpra#3371]
```
$ xpra detach
xpra initialization error:
 there are multiple servers running, please specify
```
When `xpra` is run from an xpra-driven window, it should be reasonable to assume that the server in question is $DISPLAY, yes? Is there a good reason not to default to $DISPLAY?

Many subcommands could assume this, like `xpra screenshot xyz.png`. Perhaps something more precise and xpra-specific than $DISPLAY is in order, like maybe `XPRA_SERVER_SOCKET`.

When there's a genuine ambiguity, notably when no $DISPLAY exists, sure, xpra should ask for specifics.

**Comment by totaam** [comment]
Done in the commit above for most subcommands: `info`, `id`, `shell`, `qrcode`, `version`, `detach`, `show-menu`, `show-session-info`, etc..
This uses `XPRA_SERVER_SOCKET` because, as you pointed out, that's much more likely to be correct.

**Comment by normanr** [comment]
I noticed that `control` and `print` both still _require_ the `DISPLAY` argument, but I discovered that it's allowed to be empty :-)

**Comment by totaam** [comment]
> I noticed that `control` and `print` both still require the DISPLAY argument, but I discovered that it's allowed to be empty :-)

Yes, we use positional arguments so it can't be removed completely but you can use an empty placeholder.
